include/reloader.h

```diff
@@ -24,7 +24,7 @@ class Reloader {
 	/// indexes.
 	///
 	/// If \a indexes is nullptr, all feeds will be reloaded.
-	void start_reload_all_thread(std::vector<int>* indexes = nullptr);
+	void start_reload_all_thread(const std::vector<int>& indexes = {});
```

Comment:
Please update the comment, it shouldn't mention `nullptr`.

Reply:
Oh, thanks, I didn't notice this! I'll fix it later today :)